src/shape/fractal/HilbertEncoder.cpp

```diff
@@ -65,7 +65,7 @@ HilbertEncoder::sort(std::vector<geom::Geometry*>& geoms)
         bool
         operator()(const geom::Geometry* a, const geom::Geometry* b)
         {
-            return enc.encode(a->getEnvelopeInternal()) > enc.encode(b->getEnvelopeInternal());
+            return enc.encode(a->getEnvelopeInternal()) < enc.encode(b->getEnvelopeInternal());
```

Comment:
@pramsey I changed the sort direction to better match expected results; `std::sort` returns items in ascending order, and the few other places I checked in GEOS that were sorting objects with a comparator like this were also doing so in ascending order.  Presuming the descending Hilbert sort order was not intentional, of course.

I didn't see any other usage of this function in GEOS.  Does this merit a note in the `NEWS`?

Reply:
Was only changing out of interest of consistency / expected results; since this PR added tests based on expected behavior, seemed like a good time to do so.

Definitely don't want to piss anyone off, just thought it was odd that a sort that normally would produce ascending order did the opposite here.

Would you like me to revert this change?

Reply:
Yes, I think so. One of the privileges of being a widely used library is being stuck with your history, particularly in the case of behaviour.

Reply:
The new C API function could do an ascending sort, no?

Reply:
True